Dockerfile

```diff
@@ -2,15 +2,11 @@ FROM jupyter/base-notebook:lab-3.2.9
 
 USER root
 
-COPY . /opt/cs3/
+COPY dist /opt/cs3/
```

Comment:
what do you mean?

Reply:
do we really need to build this outside of the pipeline?

Reply:
Yeah we build images for testing https://hub.docker.com/r/cs3mesh/cs3apis4lab. And it would be kinda bad to have a Dockerfile that only works inside github. If you really want to keep this you can create an additional Dockerfile and builld with docker build . --file <dockerfile-for-github>

Reply:
It works outside of github. As I said, you need to build the extension outside. But I'll revert this and keep as it was. Gonna merge the PR.